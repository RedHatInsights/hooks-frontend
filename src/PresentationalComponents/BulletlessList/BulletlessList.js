import React from 'react';
import { List } from '@patternfly/react-core';
import PropTypes from 'prop-types';

export const BulletlessList = ({ children }) =>
    <List style={ { listStyle: 'none' } }>
        { children }
    </List>;

BulletlessList.propTypes = {
    children: PropTypes.node
};

export default BulletlessList;
