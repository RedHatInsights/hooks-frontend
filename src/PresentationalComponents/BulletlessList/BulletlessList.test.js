import toJson from 'enzyme-to-json';

import { ListItem } from '@patternfly/react-core';
import { BulletlessList } from './BulletlessList';

describe('FilterList', () => {
    it('expect to render', () => {
        const wrapper = shallow(
            <BulletlessList>
                <ListItem>Something</ListItem>
                <ListItem>Something else</ListItem>
            </BulletlessList>
        );

        expect(toJson(wrapper.render())).toMatchSnapshot();
    });
});
