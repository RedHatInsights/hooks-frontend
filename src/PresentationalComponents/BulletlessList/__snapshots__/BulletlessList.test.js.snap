// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterList expect to render 1`] = `
<ul
  class="pf-c-list"
  style="list-style:none"
>
  <li>
    Something
  </li>
  <li>
    Something else
  </li>
</ul>
`;
